test/e2e-tests.sh

```diff
@@ -78,11 +78,6 @@ wait_until_cluster_up
 
 header "Running tests"
 kubectl create namespace serving-tests
-options=""
-(( EMIT_METRICS )) && options="-emitmetrics"
-report_go_test \
-  -v -tags=e2e -count=1 -timeout=20m \
-  ./test/conformance ./test/e2e \
-  ${options} || fail_test
+go_test_e2e -timeout=20m ./test/conformance ./test/e2e || fail_test
```

Comment:
I don't understand how this change is relevant to your PR?

Reply:
This came from a merge.